frontend/src/pages/AdminPanel/components/AdminMenu.jsx

```diff
@@ -30,7 +32,16 @@ export default function AdminPanel() {
           type="button"
           className="category-menu"
         >
-          Modifier
+          <Link
+            style={{ textDecoration: "none" }}
+            to={
+              location.pathname === "/admin/videos/modify"
+                ? "/admin"
+                : "/admin/videos/modify"
```

Comment:
Link redirect to only one url, not a dynamic one.

Reply:
This is to avoid that if you click again on the modify button you create a rewrite of the url a second time (ex: /admin/videos/modify/videos/modify) in the request